clang/include/clang/Sema/DeclSpec.h

```diff
@@ -380,7 +375,8 @@ class DeclSpec {
   unsigned FS_noreturn_specified : 1;
 
   // friend-specifier
-  unsigned Friend_specified : 1;
+  unsigned FriendSpecified : 1;
```

Comment:
Should these have that debug-attribute that we added not long ago?  I think it was `Preferred_Type` or something?

Reply:
`LLVM_PREFERRED_TYPE` I believe

Reply:
I think you're right, thats the one!